Sources/UIKit/Renderers/SpinnerRenderer.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  SpinnerRenderer.swift
+//  PortalView
+//
+//  Created by Cristian Ames on 4/21/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+
+import UIKit
+
+internal struct SpinnerRenderer<MessageType>: UIKitRenderer {
+    
+    let isActive: Bool
+    let style: StyleSheet<SpinnerStyleSheet>
+    let layout: Layout
+    
+    func render(with layoutEngine: LayoutEngine, isDebugModeEnabled: Bool) -> Render<MessageType> {
+        let spinner = UIActivityIndicatorView.init(activityIndicatorStyle: .gray)
+        
+        spinner.hidesWhenStopped = false
+        isActive ? spinner.startAnimating() : spinner.stopAnimating()
```

Comment:
`?:` should be used for its result throw a warning here.

```swift
if isActive {
  spinner.startAnimating()
} else {
  spinner.stopAnimating()
}
```

More lines but the intent is clearer